attestation-service/verifier/src/se/ibmse.rs

```diff
@@ -274,13 +279,13 @@ impl SeVerifierImpl {
                 );
                 let skip_certs: bool = skip_certs_env.parse::<bool>().unwrap_or(false);
                 if !skip_certs {
-                    let verifier = CertVerifier::new(ca_certs.as_slice(), crls.as_slice(), Some(root_ca_path.clone()), false)?;
+                    let verifier = CertVerifier::new(ca_certs.as_slice(), crls.as_slice(), ca_option.clone(), true)?;
```

Comment:
- we should better use `false` here, which mean we will download crls for every provided `ca_certs` automatically .
- otherwise we MUST not only provide `ibm-z-host-key-gen2.crl`, but also need provide crls for `DigiCertCA.crt` and `ibm-z-host-key-signing-gen2.crt`, that will be:
    - http://crl3.digicert.com/DigiCertTrustedRootG4.crl (for DigiCertCA.crt)
    - http://crl3.digicert.com/DigiCertTrustedG4CodeSigningRSA4096SHA3842021CA1.crl  (for ibm-z-host-key-signing-gen2.crt)
    - the urls can be found via commands:
        - openssl x509 -in DigiCertCA.crt --text --noout |grep crl
        - openssl x509 -in ibm-z-host-key-signing-gen2.crt --text --noout |grep crl

Reply:
Make it configurable in the new commit, thanks @liudalibj